Source/SwiftLintFramework/Rules/MarkRule.swift

```diff
@@ -24,11 +24,14 @@ public struct MarkRule: CorrectableRule, ConfigurationProviderRule {
     public static let description = RuleDescription(
         identifier: "mark",
         name: "Mark",
-        description: "MARK comment should be in valid format.",
+        description: "MARK comment should be in valid format. e.g. // MARK: ok",
```

Comment:
I don't really like this change because there're other valid formats (e.g. with a `-`).